monolith/forms.py

```diff
@@ -16,3 +16,14 @@ class UserForm(FlaskForm):
     password = f.PasswordField("password", validators=[DataRequired()])
     dateofbirth = f.DateField("dateofbirth", format="%d/%m/%Y")
     display = ["email", "firstname", "lastname", "password", "dateofbirth"]
+
+class RestaurantForm(FlaskForm):
```

Comment:
This was already implemented in https://github.com/GreyTeam2020/GoOutSafe_Primer2020/pull/5

Can you remove this change inside this file? I will join my change with yours